DiscountService.cs

```diff
@@ -0,0 +1,21 @@
+﻿namespace Price_Calculator_kata
+{
+    public class DiscountService
```

Comment:
let's add an interface that this service implements, it should contain the methods signature that you already implemented here